src/plugins/notebook/res/templates/notebook.html

```diff
@@ -31,8 +31,9 @@
     <div class="c-notebook__entries" ng-mouseover="handleActive()">
         <ul>
             <notebook-entry
-                v-for="entry in filteredAndSortedEntries"
-                v-bind:entry="entry">
+                v-for="(entry,index) in filteredAndSortedEntries"
+                :key="index"
```

Comment:
Should use `entry.key` instead of `index` here. `index` can't be relied on to uniquely identify a row - it's  an ordinal, and as such will depend on the ordering of the collection.